neon_data_models/enum.py

```diff
@@ -38,9 +38,11 @@ class AccessRoles(IntEnum):
     NONE = 0
     GUEST = 1
     USER = 2
-    ADMIN = 3
-    OWNER = 4
-
+    # 3-5 Reserved for "premium users"
+    ADMIN = 6
+    # 7-8 Reserved for "restricted owners"
+    OWNER = 9
+    # 10 Reserved for "unlimited access"
```

Comment:
10 seems like a fairly small number, might want to consider something larger like 50 or 100 for "unlimited access". Food for thought, since you haven't defined it here it's not necessarily relevant for this PR

Reply:
I was considering that too.. My main concern is that a very large range could make troubleshooting permissions more complicated, though restricting it to enumerated values does obviate that for the most part